core/embed/firmware/main.c

```diff
@@ -300,6 +300,10 @@ void SVC_C_Handler(uint32_t *stack) {
       // raising privileges.
       stack[6] = (uintptr_t)reboot_to_bootloader;
       return;
+    case SVC_GET_SYSTICK_VAL: {
+      uint32_t *ret_ptr = (uint32_t *)stack[0];
```

Comment:
this looks insecure? allows unprivileged code to write to any address.
maybe we could return the value in some register or something?

Reply:
(alternately just reserve a `static uint32_t` at a known location and the caller will need to take the systick val out of there?)

Reply:
going with the static idea: https://github.com/trezor/trezor-firmware/pull/3303/commits/8f7ae5380721fe586edadfc4e0272bbe5d8b1fb3